front/lib/api/assistant/global_agents.ts

```diff
@@ -17,6 +18,9 @@ enum GLOBAL_AGENTS_SID {
   CLAUDE_INSTANT = "claude-instant-1",
```

Comment:
Assuming the order is preserved through your new appraoch, let's move GPT4 before GPT3.5 and CLAUDE before CLAUDE-INSTANT

Reply:
I don't know if the order will be preserved. Lets just revert to the previous version. Too much time spent on that.